### PR TITLE
feat(firebase_ui_localizations): Further German translations for auth labels that were still in English

### PR DESCRIPTION
**Pull request by fighterii**
## Description

Corrected German Translations for Email Verification (which were previously not translated and just in english).

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for _all_ changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] All unit tests pass (`melos run test:unit:all` doesn't fail).
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is _not_ a breaking change.

[issue database]: https://github.com/firebase/FirebaseUI-Flutter/issues
[Contributor Guide]: https://github.com/firebase/FirebaseUI-Flutter/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by russellwheatley** [comment]
@fighterii - Thanks for the PR 😄. Do you have a fellow developer who is fluent in German to double check the translations, please?

**Comment by fighterii** [comment]
@russellwheatley  Unfortunately not. Maybe we have to trust AI here for double-checking :D :D

**Comment by russellwheatley** [comment]
Hey @nilsreichardt - what do you think of these German translations? 😄

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 1]
```diff
@@ -424,47 +424,47 @@
...
     "placeholders": {}
   },
-  "uploadButtonText": "Upload file",
+  "uploadButtonText": "Datei hochladen",
   "@uploadButtonText": {
     "description": "UploadButton label",
     "placeholders": {}
   },
-  "verifyEmailTitle": "Verify your email",
+  "verifyEmailTitle": "Bestätigen Sie Ihre Email",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:432]:
```suggestion
  "verifyEmailTitle": "Bestätigen Sie Ihre E-Mail",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 2]
```diff
@@ -424,47 +424,47 @@
...
   "@verifyEmailTitle": {
     "description": "EmailVerificationScreen title",
     "placeholders": {}
   },
-  "verificationEmailSentText": "A verification email has been sent to your email address. Please check your email and click on the link to verify your email address.",
+  "verificationEmailSentText": "Eine Bestätigungs Email wurde an Ihre Email Adresse versendet. Bitte prüfen Sie die Email und klicken auf den Link zum Bestätigen.",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:437]:
```suggestion
  "verificationEmailSentText": "Eine Bestätigungsmail wurde an Ihre E-Mail-Adresse versendet. Bitte prüfen Sie die E-Mails und klicken Sie auf den Link zum Bestätigen.",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 3]
```diff
@@ -424,47 +424,47 @@
...
   "@verificationEmailSentText": {
     "description": "Hint text indicating that verification email has been sent",
     "placeholders": {}
   },
-  "verificationFailedText": "We couldn't verify your email address. ",
+  "verificationFailedText": "Wir konnten Ihre Email Adresse nicht bestätigen.",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:442]:
```suggestion
  "verificationFailedText": "Wir konnten Ihre E-Mail-Adresse nicht bestätigen.",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 4]
```diff
@@ -424,47 +424,47 @@
...
   "@verificationFailedText": {
     "description": "Message indicating that something went wrong during email verification",
     "placeholders": {}
   },
-  "resendVerificationEmailButtonLabel": "Resend verification email",
+  "resendVerificationEmailButtonLabel": "Sende Bestätigungs Email erneut",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:447]:
```suggestion
  "resendVerificationEmailButtonLabel": "Sende Bestätigungsmail erneut",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 5]
```diff
@@ -424,47 +424,47 @@
...
   "@resendVerificationEmailButtonLabel": {
     "description": "Button label that suggests to resend verification email",
     "placeholders": {}
   },
-  "verificationEmailSentTextShort": "Verification email sent",
+  "verificationEmailSentTextShort": "Bestätigungs Email gesendet",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:452]:
```suggestion
  "verificationEmailSentTextShort": "Bestätigungsmail gesendet",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 6]
```diff
@@ -424,47 +424,47 @@
...
   "@verificationEmailSentTextShort": {
     "description": "Short version of the hint text indicating that verification email has been sent",
     "placeholders": {}
   },
-  "emailIsNotVerifiedText": "Email is not verified",
+  "emailIsNotVerifiedText": "Email ist noch nicht bestätigt",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:457]:
```suggestion
  "emailIsNotVerifiedText": "E-Mail ist noch nicht bestätigt",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 7]
```diff
@@ -424,47 +424,47 @@
...
   "@emailIsNotVerifiedText": {
     "description": "Message indicating that email is not verified",
     "placeholders": {}
   },
-  "waitingForEmailVerificationText": "Waiting for email verification",
+  "waitingForEmailVerificationText": "Warte auf Email Bestätigung",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:462]:
```suggestion
  "waitingForEmailVerificationText": "Warte auf E-Mail-Bestätigung",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 8]
```diff
@@ -474,62 +474,62 @@
     "description": "OK button label",
     "placeholders": {}
   },
-  "checkEmailHintText": "Please check your email and click the link to verify your email address.",
+  "checkEmailHintText": "Bitte prüfen Sie Ihre Emails und klicken den Link um die Email Adresse zu bestätigen.",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:477]:
```suggestion
  "checkEmailHintText": "Bitte prüfen Sie Ihre E-Mails und klicken Sie auf den Link, um die E-Mail-Adresse zu bestätigen.",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 9]
```diff
@@ -474,62 +474,62 @@
...
     "placeholders": {}
   },
-  "weakPasswordErrorText": "Password should be at least 6 characters",
+  "weakPasswordErrorText": "Passwort muss mind. 6 Zeichen enthalten",
   "@weakPasswordErrorText": {
     "description": "Error text suggesting that used password is too weak",
     "placeholders": {}
   },
-  "confirmDeleteAccountAlertTitle": "Confirm account deletion",
+  "confirmDeleteAccountAlertTitle": "Bestätige Account Löschung",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:517]:
```suggestion
  "confirmDeleteAccountAlertTitle": "Bestätige Kontolöschung",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 10]
```diff
@@ -474,62 +474,62 @@
...
   "@confirmDeleteAccountAlertTitle": {
     "description": "Delete account confirmation dialog title",
     "placeholders": {}
   },
-  "confirmDeleteAccountAlertMessage": "Are you sure you want to delete your account?",
+  "confirmDeleteAccountAlertMessage": "Sind Sie sicher, dass Sie den Account löschen möchten?",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:522]:
```suggestion
  "confirmDeleteAccountAlertMessage": "Sind Sie sicher, dass Sie das Konto löschen möchten?",
```

File: packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb [hunk 11]
```diff
@@ -474,62 +474,62 @@
...
     "placeholders": {}
   },
-  "confirmDeleteAccountButtonLabel": "Yes, delete",
+  "confirmDeleteAccountButtonLabel": "Ja, löschen",
   "@confirmDeleteAccountButtonLabel": {
     "description": "Confirm delete account button label",
     "placeholders": {}
   },
-  "sendVerificationEmailLabel": "Send verification email",
+  "sendVerificationEmailLabel": "Sende Verifikations-Email",
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/l10n/firebase_ui_de.arb:532]:
```suggestion
  "sendVerificationEmailLabel": "Verifizierungs-E-Mail senden",
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 12]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
       "Das Passwort ist ungültig oder der Nutzer hat kein Passwort";
 
   @override
-  String get uploadButtonText => "Upload file";
+  String get uploadButtonText => "Datei hochladen";
 
   @override
-  String get verifyEmailTitle => "Verify your email";
+  String get verifyEmailTitle => "Bestätigen Sie Ihre Email";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:274]:
```suggestion
  String get verifyEmailTitle => "Bestätigen Sie Ihre E-Mail";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 13]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
   String get verificationEmailSentText =>
-      "A verification email has been sent to your email address. Please check your email and click on the link to verify your email address.";
+      "Eine Bestätigungs Email wurde an Ihre Email Adresse versendet. Bitte prüfen Sie die Email und klicken auf den Link zum Bestätigen.";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:278]:
```suggestion
      "Eine Bestätigungsmail wurde an Ihre E-Mail-Adresse versendet. Bitte prüfen Sie die Emails und klicken Sie auf den Link zum Bestätigen.";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 14]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
   String get verificationFailedText =>
-      "We couldn't verify your email address. ";
+      "Wir konnten Ihre Email Adresse nicht bestätigen.";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:282]:
```suggestion
      "Wir konnten Ihre E-Mail-Adresse nicht bestätigen.";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 15]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
-  String get resendVerificationEmailButtonLabel => "Resend verification email";
+  String get resendVerificationEmailButtonLabel =>
+      "Sende Bestätigungs Email erneut";
 
   @override
-  String get verificationEmailSentTextShort => "Verification email sent";
+  String get verificationEmailSentTextShort => "Bestätigungs Email gesendet";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:289]:
```suggestion
  String get verificationEmailSentTextShort => "Bestätigungsmail gesendet";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 16]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
-  String get emailIsNotVerifiedText => "Email is not verified";
+  String get emailIsNotVerifiedText => "Email ist noch nicht bestätigt";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:292]:
```suggestion
  String get emailIsNotVerifiedText => "E-Mail ist noch nicht bestätigt";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 17]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
-  String get dismissButtonLabel => "Dismiss";
+  String get dismissButtonLabel => "Verwerfen";
 
   @override
   String get okButtonLabel => "OK";
 
   @override
   String get checkEmailHintText =>
-      "Please check your email and click the link to verify your email address.";
+      "Bitte prüfen Sie Ihre Emails und klicken den Link um die Email Adresse zu bestätigen.";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:305]:
```suggestion
      "Bitte prüfen Sie Ihre E-Mails und klicken Sie auf den Link, um die E-Mail-Adresse zu bestätigen.";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 18]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
+      "Sind Sie sicher, dass Sie diesen Provider trennen möchten?";
 
   @override
-  String get weakPasswordErrorText =>
-      "Password should be at least 6 characters";
+  String get weakPasswordErrorText => "Passwort muss mind. 6 Zeichen enthalten";
 
   @override
-  String get confirmDeleteAccountAlertTitle => "Confirm account deletion";
+  String get confirmDeleteAccountAlertTitle => "Bestätige Account Löschung";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:331]:
```suggestion
  String get confirmDeleteAccountAlertTitle => "Bestätige Kontolöschung";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 19]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
   String get confirmDeleteAccountAlertMessage =>
-      "Are you sure you want to delete your account?";
+      "Sind Sie sicher, dass Sie den Account löschen möchten?";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:335]:
```suggestion
      "Sind Sie sicher, dass Sie das Konto löschen möchten?";
```

File: packages/firebase_ui_localizations/lib/src/lang/de.dart [hunk 20]
```diff
@@ -268,76 +268,75 @@ class DeLocalizations extends FirebaseUILocalizationLabels {
...
 
   @override
-  String get confirmDeleteAccountButtonLabel => "Yes, delete";
+  String get confirmDeleteAccountButtonLabel => "Ja, löschen";
 
   @override
-  String get sendVerificationEmailLabel => "Send verification email";
+  String get sendVerificationEmailLabel => "Sende Verifikations-Email";
```
Comment (nilsreichardt) [inline packages/firebase_ui_localizations/lib/src/lang/de.dart:341]:
```suggestion
  String get sendVerificationEmailLabel => "Verifizierungs-E-Mail senden";
```

**Review by nilsreichardt — APPROVED** [review]
Besides the suggestions, LGTM

**Comment by russellwheatley** [comment]
@nilsreichardt - Thank you!

**Pull request merged**